python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -82,14 +88,22 @@ def test_sets(self) -> List[ItemSet] or List[ItemSetDict]:
         """Return the test set."""
         return self._test_sets
 
-    def graph(self) -> object:
+    def graph(self) -> CSCSamplingGraph:
         """Return the graph."""
-        raise NotImplementedError
+        return self._graph
 
     def feature(self) -> Dict[Tuple, TorchBasedFeatureStore]:
         """Return the feature."""
         return self._feature
 
+    def _load_graph(
+        self, graph_topology: OnDiskGraphTopology
+    ) -> CSCSamplingGraph:
+        """Load the graph topology."""
+        if graph_topology is None:
+            return None
+        return load_csc_sampling_graph(graph_topology.path)
```

Comment:
In yaml, we have type: CSCSamplingGraph, we should check the type first, such that in the future when we support new type, the old dgl version can failure properly.

Reply:
when we support new type, the old one(type) is supposed to work well?
pydantic is helping us verify the enum value automatically.

Reply:
I see, the I have less concern. nit: still worth to check the OnDiskGraphTopologyType = CSC_SAMPLING before using load_csc_sampling_graph(graph_topology.path), and return none for others.

Reply:
sure.